freqtrade/freqtradebot.py

```diff
@@ -204,19 +204,12 @@ def _process(self) -> bool:
             self.state = State.STOPPED
         return state_changed
 
-    def get_target_bid(self, pair: str, ticker: Dict[str, float]) -> float:
+    def get_target_bid(self, pair: str, test_ticker: Dict[str, float] = None) -> float:
```

Comment:
It makes no sense to me to leave test_ticker in there just to make tests happy, since then they're testing nothing relevant.

for the relevant tests, `exchange.get_ticker` should be mocked with the result, so we can remove this parameter.